nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -212,9 +213,30 @@ in {
       };
 
       localDomain = lib.mkOption {
-        description = lib.mdDoc "The domain serving your Mastodon instance.";
+        description = lib.mdDoc ''
```

Comment:
After reading both documentation strings, it is still not clear to me from first glance what is what.

I think the main problem I had comes from both first sentences being very similar. How about "The domain that is going to show up in usernames" and "The domain serving the Mastodon web interface." as first sentences?